services/ux-backend/main.go

```diff
@@ -63,6 +67,21 @@ func main() {
 		os.Exit(-1)
 	}
 
+	namespace := util.GetPodNamespace()
```

Comment:
we are starting w/ supporting a single storagecluster that exists in operator ns?

Reply:
Yes